Maple2Storage/Scripts/Npcs/11000508.lua

```diff
@@ -8,25 +8,25 @@ function getBeginEventId(itemRarity, enchantLevel)
             return 1001
         end
     end
-    return 1998 + enchantLevel
+    return 1198 + enchantLevel
```

Comment:
i dislike the idea of hardcode numbers, but aparently is the best option right now

Reply:
its not hardcoded numbers, its the the event ids from the XMLs.